### PR TITLE
Prove theorems without ax-8

### PR DESCRIPTION
**Pull request by GinoGiotto**
Edit a few proofs to reduce axiom usage, specifically:

* Edit [abv](https://us.metamath.org/mpeuni/abv.html) -> avoid ax-8
* Edit [rru](https://us.metamath.org/mpeuni/rru.html) -> avoid ax-10, ax-11, ax-12
* Edit [ab0](https://us.metamath.org/mpeuni/ab0.html) -> avoid ax-8
* Edit [ab0orv](https://us.metamath.org/mpeuni/ab0orv.html) -> avoid ax-10, ax-11, ax-12 (ax-8 automatically avoided by previous edits)
* Edit [abn0](https://us.metamath.org/mpeuni/abn0.html) -> avoid ax-8

All added DV conditions are with dummy variables.

Axiom usage here: https://github.com/metamath/set.mm/commit/781a0bb89850b0271ab3b6d5288a598a6f318f57

### CODE REVIEW

**Review by icecream17 — APPROVED** [review]

**Review by benjub — APPROVED** [review]
Nice axiom reduction!

And your PR comment makes it very easy to review.

**Comment by benjub** [comment]
A shortening:
```
    abv $p |- ( { x | ph } = _V <-> A. x ph ) $=
      ( vy cab wtru wceq wsb wal cvv cv wcel wb dfcleq vextru tbt df-clab albii
      bitr3i bitri weq df-v equid bitru abbii eqtri eqeq2i nfv sb8v 3bitr4i ) A
      BDZEBDZFZABCGZCHZUJIFABHULCJZUJKZUOUKKZLZCHUNCUJUKMURUMCURUPUMUQUPBCNOACB
      PRQSIUKUJIBBTZBDUKBUAUSEBUSBUBUCUDUEUFABCACUGUHUI $.
```
and this is one more reason to use https://us.metamath.org/mpeuni/bj-df-v.html as the definition (and this theorem does not require ax-8 nor df-clel either).

File: set.mm [hunk 1]
```diff
@@ -31672,9 +31672,24 @@ generally appear in a single form (either definitional, but more often
     $d y x $.  $d y ph $.
     $( The class of sets verifying a property is the universal class if and
        only if that property is a tautology.  The reverse implication
-       ( ~ bj-abv ) requires fewer axioms.  (Contributed by BJ,
-       19-Mar-2021.) $)
+       ( ~ bj-abv ) requires fewer axioms.  (Contributed by BJ, 19-Mar-2021.)
+       Avoid ~ ax-8 .  (Revised by Gino Giotto, 30-Aug-2024.) $)
```
[the remark this answers is not in the document]
Comment (GinoGiotto) [reply set.mm:31676]: It does, added in the comment in $j.

File: set.mm [hunk 2]
```diff
@@ -31672,9 +31672,24 @@ generally appear in a single form (either definitional, but more often
...
+  $}
+
+  ${
+    $d y x $.  $d y ph $.
+    $( Obsolete version of ~ abv as of 30-Aug-2024.  (Contributed by BJ,
+       19-Mar-2021.)  (Proof modification is discouraged.)
+       (New usage is discouraged.) $)
+    abvOLD $p |- ( { x | ph } = _V <-> A. x ph ) $=
       ( vy cv cab wcel wal wsb cvv wceq df-clab albii eqv nfv sb8v 3bitr4i ) CD
       ABEZFZCGABCHZCGQIJABGRSCACBKLCQMABCACNOP $.
```
[the remark this answers is not in the document]
Comment (GinoGiotto) [reply set.mm]: Done

File: set.mm [hunk 3]
```diff
@@ -39915,12 +39947,28 @@ among classes ( ~ eq0 , as opposed to the weaker uniqueness among sets,
...
+      FUGCVBVCUKVHBCLZAMZBJZGIZCJVOKZCJVJVGVPCVEVOVFGVEABCNVOACBOABCUHPVFGBCNGG
+      CBOGBCUIPUJQVPVQCVQVPVOULUMQVQVICBVNBUNVICUOCBLZVOAVRVOAVOAMBCVMVNABVMAUP
+      UQRAVOMBCABCURRUSUTVATT $.
+    $( $j usage 'ab0' avoids 'ax-8'; $)
+  $}
+
+  $( Obsolete version of ~ ab0 as of 30-Aug-2024.  (Contributed by BJ,
+     19-Mar-2021.)  (Proof modification is discouraged.)
+     (New usage is discouraged.) $)
+  ab0OLD $p |- ( { x | ph } = (/) <-> A. x -. ph ) $=
```
[the remark this answers is not in the document]
Comment (GinoGiotto) [reply set.mm]: Done

File: set.mm [hunk 4]
```diff
@@ -39915,12 +39947,28 @@ among classes ( ~ eq0 , as opposed to the weaker uniqueness among sets,
...
-     at the cost of depending on ~ df-ne ).  (Contributed by BJ,
-     19-Mar-2021.) $)
-  ab0 $p |- ( { x | ph } = (/) <-> A. x -. ph ) $=
+  ${
+    $d x y $.  $d y ph $.
+    $( The class of sets verifying a property is the empty class if and only if
+       that property is a contradiction.  See also ~ abn0 (from which it could
+       be proved using as many essential proof steps but one fewer syntactic
+       step, at the cost of depending on ~ df-ne ).  (Contributed by BJ,
+       19-Mar-2021.)  Avoid ~ ax-8 .  (Revised by Gino Giotto, 30-Aug-2024.) $)
```
[the remark this answers is not in the document]
Comment (GinoGiotto) [reply set.mm:39956]: Done

File: set.mm [hunk 5]
```diff
@@ -39931,18 +39979,45 @@ among classes ( ~ eq0 , as opposed to the weaker uniqueness among sets,
...
     ab0orv $p |- ( { x | ph } = _V \/ { x | ph } = (/) ) $=
+      ( vy wal wn wo cab cvv wceq c0 mpbi cv wcel weq wsb df-clab bitr2i sbievw
+      wb bibi12i wnf nfv nf3 equid tbt sbv id eqeq12d bitri albii dfcleq eqcomi
+      df-v eqeq2i 3bitr2i nbn3 notbid dfnul2 orbi12i ) ACDZAEZCDZFZABGZHIZVDJIZ
+      FACUAVCACUBACUCKUTVEVBVFUTCLZVDMZVGBBNZBGZMZSZCDVDVJIVEAVLCAACCNZSVLVMACU
+      DZUEAVHVMVKVHABCOAACBPABCUFQZVKVIBCOVMVICBPVIVMBCBCNZBLZVGVQVGVPUGZVRUHZR
+      QTUIUJCVDVJUKVJHVDHVJBUMULUNUOVBVHVGVIEZBGZMZSZCDVDWAIVFVAWCCVAAVMEZSWCVM
+      AVNUPAVHWDWBVOWBVTBCOWDVTCBPVTWDBCVPVIVMVSUQRQTUIUJCVDWAUKWAJVDJWABURULUN
+      UOUSK $.
+    $( $j usage 'ab0orv' avoids 'ax-8' 'ax-10' 'ax-11' 'ax-12'; $)
```
[the remark this answers is not in the document]
Comment (GinoGiotto) [reply set.mm:39997]: Done

File: set.mm [hunk 6]
```diff
@@ -39931,18 +39979,45 @@ among classes ( ~ eq0 , as opposed to the weaker uniqueness among sets,
...
+  $}
+
+  ${
+    $d x ph $.
+    $( Obsolete version of ab0orv as of 30-Aug-2024.  (Contributed by Mario
+       Carneiro, 29-Aug-2013.)  (Revised by BJ, 22-Mar-2020.)
+       (Proof modification is discouraged.)  (New usage is discouraged.) $)
+    ab0orvOLD $p |- ( { x | ph } = _V \/ { x | ph } = (/) ) $=
```
[the remark this answers is not in the document]
Comment (GinoGiotto) [reply set.mm:40005]: Done

File: set.mm [hunk 7]
```diff
@@ -39931,18 +39979,45 @@ among classes ( ~ eq0 , as opposed to the weaker uniqueness among sets,
...
       ( wnf cab cvv wceq c0 wo nfv dfnf5 mpbi ) ABCABDZEFLGFHABIABJK $.
   $}
 
   $( Nonempty class abstraction.  See also ~ ab0 .  (Contributed by NM,
-     26-Dec-1996.)  (Proof shortened by Mario Carneiro, 11-Nov-2016.) $)
+     26-Dec-1996.)  (Proof shortened by Mario Carneiro, 11-Nov-2016.)  Avoid
+     ~ ax-8 .  (Revised by Gino Giotto, 30-Aug-2024.) $)
```
[the remark this answers is not in the document]
Comment (GinoGiotto) [reply set.mm:40011]: Done

**Comment by GinoGiotto** [comment]
> and this is one more reason to use [us.metamath.org/mpeuni/bj-df-v.html](https://us.metamath.org/mpeuni/bj-df-v.html)

I definitely agree, I often use `T.` for tautologies and it generally makes proofs simpler. I would also argue to use [us.metamath.org/mpeuni/bj-df-nul.html](https://us.metamath.org/mpeuni/bj-df-nul.html) for the same reason.

**Review by tirix — APPROVED** [review]

**Review by benjub — APPROVED** [review]

**Pull request merged**